.github/fabricbot.json

```diff
@@ -444,48 +429,47 @@
             "area-System.Memory"
           ],
           "mentionees": [
-            "GrabYourPitchForks",
-            "carlossanlop"
+            "dotnet/area-system-memory"
```

Comment:
I think you should keep @GrabYourPitchforks explicitly indicated here. He recently asked to be added to the notifications, even though he's not an area owner.

Reply:
I confirmed that @GrabYourPitchforks is part of the memory team in GitHub.